web/app/themes/mitlib-parent/functions.php

```diff
@@ -170,8 +170,7 @@ function setup_scripts_styles() {
 	wp_register_script( 'parent-production', get_template_directory_uri() . '/js/alerts.js', array( 'dev', 'fonts', 'jquery', 'libchat', 'menu-toggle' ), $theme_version, true );
 
 	// Interior bundle.
-	wp_register_script( 'ga-links', get_template_directory_uri() . '/js/ga_links.js', array(), $theme_version, true );
-	wp_register_script( 'parent-interior', get_template_directory_uri() . '/js/core.js', array( 'ga-links' ), $theme_version, true );
+	wp_register_script( 'parent-interior', get_template_directory_uri() . '/js/core.js', $theme_version, true );
```

Comment:
@jazairi I think this is still missing an argument still and needs array() added like:
```php
wp_register_script( 'parent-interior', get_template_directory_uri() . '/js/core.js', array() , $theme_version, true );
```

Reply:
Agreed - the empty array is meaningful to wp_register_script in this case. I believe (but haven't been able to completely verify this because of my Lando issues) that you can see the difference at a URL like this:

https://uxws-1555-mitlib-wp-network.pantheonsite.io/study/group/

The location panels on this page should be expandable, but aren't working here (any other page with this UI element should be similarly affected - this is just one I know off-hand)

Reply:
Thanks @JPrevost for pointing that out, and @matt-bernhardt for the context. I just pushed a fix here and to the multidev. The panels on the page you shared are now expandable.